cloud-controller-manager/instance.go

```diff
@@ -0,0 +1,69 @@
+package ccm
+
+import (
+	"context"
+	"fmt"
+
+	ctlKubevirt "github.com/harvester/harvester/pkg/generated/controllers/kubevirt.io"
+	ctlKubevirtv1 "github.com/harvester/harvester/pkg/generated/controllers/kubevirt.io/v1"
```

Comment:
prefer not to use uppercase for package name.
```
By convention, packages are given lower case, single-word names; there should be no need for underscores or mixedCaps. Err on the side of brevity, since everyone using your package will be typing that name
```